libs/sdk-ui-tests-e2e/scenarios/webpack.config.js

```diff
@@ -0,0 +1,184 @@
+        }),
+        new ForkTsCheckerWebpackPlugin({
+            issue: {
+                include: [{ file: "src/**/*.{ts,tsx}" }],
+            },
+        }),
+    ];
+
+    if (isProduction) {
+        plugins.push(new CompressionPlugin());
+    }
+
+    // flip the `disable` flag to false if you want to diagnose webpack perf
+    const smp = new SpeedMeasurePlugin({ disable: true });
+
+    return smp.wrap({
+        entry: ["./scenarios/src/index.tsx"],
+        target: "web",
+        mode: isProduction ? "production" : "development",
+        plugins,
+        output: {
+            filename: "[name].[contenthash].js",
+            path: path.join(__dirname, "build"),
+            publicPath: `./`,
+        },
+        devtool: isProduction ? false : "cheap-module-source-map",
+        node: {
+            __filename: true,
+        },
+        resolve: {
+            extensions: [".js", ".jsx", ".ts", ".tsx"],
+            alias: {
+                react: path.resolve("./node_modules/react"),
+                // fixes tilde imports in CSS from sdk-ui-ext
+                "@gooddata/sdk-ui-ext": path.resolve("./node_modules/@gooddata/sdk-ui-ext"),
+                "@gooddata/sdk-ui-kit": path.resolve("./node_modules/@gooddata/sdk-ui-kit"),
+                "@gooddata/sdk-ui-dashboard": path.resolve("./node_modules/@gooddata/sdk-ui-dashboard"),
+                "@gooddata/sdk-backend-spi": path.resolve("./node_modules/@gooddata/sdk-backend-spi"),
+                "@gooddata/sdk-ui": path.resolve("./node_modules/@gooddata/sdk-ui"),
+                "@gooddata/sdk-ui-charts": path.resolve("./node_modules/@gooddata/sdk-ui-charts"),
+                "@gooddata/sdk-ui-filters": path.resolve("./node_modules/@gooddata/sdk-ui-filters"),
+                "@gooddata/sdk-ui-geo": path.resolve("./node_modules/@gooddata/sdk-ui-geo"),
+                "@gooddata/sdk-ui-pivot": path.resolve("./node_modules/@gooddata/sdk-ui-pivot"),
+                "@gooddata/sdk-model": path.resolve("./node_modules/@gooddata/sdk-model"),
+                "@gooddata/sdk-backend-bear": path.resolve("./node_modules/@gooddata/sdk-backend-bear"),
+            },
+
+            // Prefer ESM versions of packages to enable tree shaking and easier dev experience
+            mainFields: ["module", "browser", "main"],
+        },
+        module: {
+            rules: [
+                {
+                    test: /\.css$/,
+                    use: ["style-loader", "css-loader"],
+                },
+                {
+                    test: /\.s[ac]ss$/,
+                    use: ["style-loader", "css-loader", "sass-loader"],
+                },
+                {
+                    test: /\.[jt]sx?$/,
+                    include: path.resolve(__dirname, "src"),
+                    use: ["babel-loader"],
+                },
+                {
+                    test: /\.js$/,
+                    include: (rawModulePath) => {
```

Comment:
I think it is safe to remove this rule: it is only for IE11 and we will never run this app in IE11.